recipes/magicblast/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "1.4.0" %}
+
+package:
+  name: magicblast
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.ncbi.nlm.nih.gov/blast/executables/magicblast/LATEST/ncbi-magicblast-{{ version }}-x64-linux.tar.gz # [linux]
+  url: ftp://ftp.ncbi.nlm.nih.gov/blast/executables/magicblast/LATEST/ncbi-magicblast-{{ version }}-x64-macosx.tar.gz # [macos]
+  sha256: 9995524a2ce67850516debb3341f268d485113f1c28005e39b5fbf47fe27d250 # [linux]
+  sha256: f48daeb86065be4f256820b5c189f0141eafdf50e5a0b8e837b381e82a447b0e # [macos]
```

Comment:
Any chance to build this from source?

Reply:
I am planning to update this recipe later to build from sources. There are a few dependencies that are not easy to deal with and I wanted to have this available sooner.

Reply:
promise? ;-)

Reply:
As you may know, it is *a lot* of work getting a source build to work for tools that use the BLAST build system. It took me many hours over the course of a week for IgBLAST. If under these circumstances a binary works with some minor ELF patching, then that should be good enough.

The recipe should be accepted without the condition that it be "fixed" later, especially when it comes from a first-time contributor.

Reply:
@epruesse, I promise cross my heart ... :-)

Reply:
@marcelm I am very well aware - it cost me a week to migrate the BLAST recipe to building from source. There are, however, many good reasons as to why we should build from source rather than just repackage binaries. That said, I also understand when people need a package ASAP and can't fight the compiler right now.

@boratyng Thank you! That would be greatly appreciated! :)